### PR TITLE
feat: double crafting output of throwables and lower value by about 30%

### PR DESCRIPTION
**Pull request by Uberbagel**
This primarily affects purchasing these items from the Alchemist buildings in the south. This change is based on player feedback as well as my own experience, having never bought those items at the Alchemist as they are far too expensive. Consider that the stated value in the file is still modified by buyprice modifiers of the faction/settlement.

### CODE REVIEW

**Comment by Darxo** [comment]
Counter-Suggestion: 
- Leave the prices the same
- Make it so that crafting many consumable items now always generates two of them
  - Increase blueprint cost of those recipes slightly
  
This should make these bombs and pots much more accessible and hunting down the materials more valueable. But you still have to pay a premium if you want to deterministically stock up on them, as alchemists almost always have them.

**Comment by Uberbagel** [comment]
> This is a big nerf to crafting those items. I don't know why I would now carry around the ingredients with me instead of just selling them outright now that their sell price is basically like the buy-price of the crafted thing.
> 
> Sulphurous Rocks sell for 300. Ghoul Teech sell for 30. Firebomb Recipe costs 100 Crowns. Buying the Fire Bomb now outright is 10 crowns cheaper.

I think your calculations are way off. On a fresh campaign at a completely neutral city-state with no modifiers, firepots are selling for 768 crowns.

**Comment by Darxo** [comment]
> I think your calculations are way off. On a fresh campaign at a completely neutral city-state with no modifiers, firepots are selling for 768 crowns.

You lower the base value of a fire pot down to 420. That's 10 crowns cheaper than if I were to sell its crafting ingredients at the base value.

**Comment by Uberbagel** [comment]
> > I think your calculations are way off. On a fresh campaign at a completely neutral city-state with no modifiers, firepots are selling for 768 crowns.
> 
> You lower the base value of a fire pot down to 420. That's 10 crowns cheaper than if I were to sell its crafting ingredients at the base value.

Then we can set a buypricemult on all crafted items of 0.70

**Comment by Darxo** [comment]
> Then we can set a buypricemult on all crafted items of 0.70

Same thing. You can still sell the ingredients and buy the crafted items for a similar same base price after this change.
It's basically a nerf to crafting of consumables. Before it had it's uses to horde ingredients, now it's almost never worth it.

**Review by LordMidas — APPROVED** [review]

**Review by Darxo — APPROVED** [review]

File: mod_reforged/hooks/crafting/blueprints/acid_flask_blueprint.nut [hunk 1]
```diff
@@ -0,0 +1,7 @@
+::Reforged.HooksMod.hook("scripts/items/crafting/blueprints/acid_flask_blueprint", function(q) {
+	q.onCraft = @(__original) function( _stash )
+	{
+		__original(_stash);
+		_stash.add(::new(::IO.scriptFilenameByHash(this.m.PreviewCraftable.ClassNameHash))); // crafting gives one additional yield over vanilla
```
[the remark this answers is not in the document]
Comment (Uberbagel) [reply mod_reforged/hooks/crafting/blueprints/acid_flask_blueprint.nut:5]: Smart point, thanks for mentioning.

**Pull request merged**